.gitignore

```diff
@@ -14,5 +14,8 @@ node_modules/
 npm-debug.log*
 package-lock.json
 packages/react-ui-kit/styleguide
+**/src/**/*.d.ts
+**/src/**/*.js
+**/src/**/*.js.map
```

Comment:
I think this file is sorted by name.. (or whatever the autosort of your preferred editor is doing)

Reply:
It is sorted by priority - last line is highest priority.